controlplane/control/digest.go

```diff
@@ -78,20 +70,12 @@ type Digest struct {
 	Value DigestValue
 }
 
-func (d Digest) CLIInfo() string {
```

Comment:
The idea of having these `CLIInfo()` helper functions inside the control package with the struct definitions is that when we add a field, we can easily update what we show to the user without having to go to the ctl (or maybe in the future other places) and update the message to e.g. add the new field.

I think that hacing them collocated with the structs makes it easier to not miss updating the ctl messages.

Reply:
The main reason to move that logic it was because some translations between stream uid and stream name have to be performed in neblictl, and digest just has the information about the stream uid (stream name is not needed for anything in the platform internal behavior).

Seeing that, I thought it would be better to decouple the data definition from the data representation. I have the same concern as you about updating what's shown in neblictl. But I could not find any easy solution.

Reply:
Mmh then we could embed these structs in other structs used by ctl and add whatever additional methods we need in there, then all the exposed messages will be in one place and the data presentation will be split from the definition.